spec.html

```diff
@@ -16961,24 +17218,28 @@ <h1>Runtime Semantics: ForIn/OfHeadEvaluation ( _TDZnames_, _expr_, _iterationKi
             1. Return ? EnumerateObjectProperties(_obj_).
           1. Else,
             1. Assert: _iterationKind_ is ~iterate~.
```

Comment:
https://github.com/tc39/ecma262/issues/1812 suggests this line should probably have been removed in this PR?